src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
> But as you can see, 'uap-aot' doesn't run in app-container.

This is temporary as pointed by @joperezr comment. so I expect we'll support running aot in app containers.